### DISCUSSION TITLE
Köln is missing

### DISCUSSION THREAD

**Discussion by LukasBaginski1** [Klizzy/MMM-Vrr#13] [Q&A] [answered]
Hi. Danke erstmal für das tolle Modul! Ich hatte es die.letzte Zeit immer laufen. Aber seit dem Update wird Köln nicht mehr unterstützt kann das sein? Modul wird nicht mehr eingeblendet wenn ich zB Köln und Hauptbahnhof eingebe

**Comment by Klizzy** [comment, marked as answer]
Hey @LukasBaginski1 ,
kannst du mir bitte den Auszug aus deiner `config.js` schicken?

Scheinbar hat sich etwas im Bezug auf die Haltestellen Namen bei der verwendeten API geändert.

Mit diesen `config.js` Einstellungen klappt es bei mir ✅:
```
        {
            module: 'MMM-Vrr',
            position: "top_right",
            config: {
                city: 'Köln',
                station: 'Hauptbahnhof Köln',
                numberOfResults: 10,
                updateInterval: 60000,
                displayIcons: true,
                displayTimeOption: 'countdown'
            }
        }
```

Mit diesen `config.js` Einstellungen klappts nicht ❌:
```
        {
            module: 'MMM-Vrr',
            position: "top_right",
            config: {
                city: 'Köln',
                station: 'Hauptbahnhof',
                numberOfResults: 10,
                updateInterval: 60000,
                displayIcons: true,
                displayTimeOption: 'countdown'
            }
        }
```

**Comment by LukasBaginski1** [comment]
hi danke für die Antwort. Komischer weise ging es 2 tage später auf einmal wieder :DAm 09.05.21, 18:50 schrieb Steven Zemelka ***@***.***>:

   
  Hey @LukasBaginski1 , kannst du mir bitte den Auszug aus deiner config.js schicken? 
  Scheinbar hat sich etwas im Bezug auf die Haltestellen Namen bei der verwendeten API geändert. 
  Mit diesen Einstellungen klappt es bei mir ✅: 
  
           {

            module: 'MMM-Vrr',

            position: "top_right",

            config: {

                city: 'Köln',

                station: 'Hauptbahnhof Köln',

                numberOfResults: 10,

                updateInterval: 60000,

                displayIcons: true,

                displayTimeOption: 'countdown'

            }

        },
   
  Mit diesen Einstellungen klappt nicht ❌: 
  
           {

            module: 'MMM-Vrr',

            position: "top_right",

            config: {

                city: 'Köln',

                station: 'Hauptbahnhof Köln',

                numberOfResults: 10,

                updateInterval: 60000,

                displayIcons: true,

                displayTimeOption: 'countdown'

            }

        },

**Reply by Klizzy** [reply]
Das freut mich zu hören!
Falls nochmal was sein sollte, mach ruhig eine Issue oder Discussion auf GitHub auf ☺️

**Comment by LukasBaginski1** [comment]
Danke wird gemacht!Am 09.05.21, 21:15 schrieb Steven Zemelka ***@***.***>:

   
  Das freut mich zu hören! Falls nochmal was sein sollte, mach ruhig eine Issue oder Discussion auf GitHub auf ☺️
